opensearch/sycamore-opensearch.sh

```diff
@@ -427,4 +433,61 @@ setup_transient() {
     deploy_model "${OPENAI_MODEL_ID}" "${OPENAI_TASK_ID}" "OpenAI"
 }
 
+create_certificates() {
+    local HOST="${SSL_HOSTNAME:-localhost}"
+    local DAYS=10000
+
+    pushd config
+    local OLDMASK=$(umask -p)
+    umask 077
+
+    # 1. Make fake certificate authority (CA) certificate.  OpenSearch
+    # requires a root certificate to be specified.
+    openssl req -batch -x509 -newkey rsa:4096 -days "${DAYS}" \
+    -subj "/C=US/ST=California/O=Aryn.ai/CN=Fake CA" \
+    -extensions v3_ca -noenc -keyout cakey.pem -out cacert.pem 2> /dev/null
```

Comment:
For now, it doesn't matter because nothing is keeping a list of accepted OpenSearch certificates, unlike browsers and HTTPS.  Indeed, the security plugin needs them to be in config.  I can give the symlinks a try; a quick web search suggests it will work.